.github/workflows/build.yml

```diff
@@ -7,6 +7,9 @@ on:
       - main
   workflow_dispatch:
 
+permissions:
+  contents: write
```

Comment:
What is this needed for?

Reply:
Couldn't pass the workflow checks on pushing without this